project/SparkBuild.scala

```diff
@@ -862,21 +862,27 @@ object SparkConnectClient {
     (assembly / assemblyPackageScala / assembleArtifact) := false,
 
     // Exclude `pmml-model-*.jar`, `scala-collection-compat_*.jar`,`jsr305-*.jar` and
-    // `netty-*.jar` and `unused-1.0.0.jar` from assembly.
+    // `unused-1.0.0.jar` and checker-qual from assembly.
     (assembly / assemblyExcludedJars) := {
       val cp = (assembly / fullClasspath).value
       cp filter { v =>
```

Comment:
Maybe change to  use `filterNot` and make the filter condition as `Include` looks clearer